static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -429,13 +450,8 @@ const makeABTestVariant = (
 
                             return component[0];
                         });
-                    });
-
-            if (test !== noop && typeof test === 'function') {
```

Comment:
This logic was allowing a `Variant` to use its `options.test` function if supplied.
This is no longer possible, and instead this function takes an optional `test` function parameter which decides what to do with the generated epic html. Otherwise it adds the epic to the page in the standard way